frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/obc-test.spec.ts

```diff
@@ -0,0 +1,140 @@
+import { expect } from 'chai';
```

Comment:
`it` blocks should be able to execute on its own. There shouldn't be dependency of one `it` block on the other.